stag-library-compiler/src/main/java/com/vimeo/stag/processor/utils/TypeUtils.java

```diff
@@ -547,7 +547,30 @@ public static String getClassNameFromTypeMirror(@NotNull TypeMirror typeMirror)
         return classAndPackage;
     }
 
+    /**
+     * Convert the provided {@link TypeMirror} into an {@link Element} instance.  This method
+     * call assumes that the provided {@link TypeMirror} is constrained to only known supported
+     * types.  As a result it will guarantee non-null result values.
+     *
+     * @param typeMirror type mirror to convert
+     * @return element representation of the type mirror
+     */
     @NotNull
+    public static Element getElementFromSupportedTypeMirror(@NotNull TypeMirror typeMirror) {
```

Comment:
Note: The original methods was annotated with `@NotNull` even though `getUtils().asElement(typeMirror)` can technically return `null` values.  The reason it didn't was only due to the fact that we only ever passed in supported types.  This method is an attempt to make this more explicit.